### ISSUE TITLE
someone with GA access to help make us GDPR complient

### ISSUE THREAD

**Issue by EwaPrzem** [findthemasks/findthemasks#527]
After some research this website suggests how we can quickly be GDPR (EUR privacy laws) compliant. For example we must  "Turn on IP Anonymization" in GA

Could someone please help fulfill these first three points outlined here:
https://www.blastanalytics.com/blog/5-actionable-steps-gdpr-compliance-google-analytics

May I also request feedback for the privacy policy regarding the data we use on GA. In plain english, what personal information do we process, how and to what end? (GPS locations, IP, anything else? to do analytics, and understand site usage, and what else)

**Comment by linderr** [comment]
@saikatbhadra ?

**Comment by saikatbhadra** [comment]
Unfortunately, I don't have bandwidth to do this right now but anyone with Google Analytics read access can do the audit (doesn't need to be a dev)

**Comment by r-pop** [comment]
I wonder if this is "good enough for now". Please reopen if not.
